pypeit/spectrographs/keck_lris.py

```diff
@@ -482,8 +429,12 @@ def default_pypeit_par(self):
         par['calibrations']['slits']['sigdetect'] = 50.
 
         # 1D wavelength solution
-        par['calibrations']['wavelengths']['lamps'] = ['ArI','NeI','HgI','KrI','XeI']
+        par['calibrations']['wavelengths']['lamps'] = ['NeI', 'ArI', 'CdI', 'KrI', 'XeI', 'ZnI', 'HgI']
```

Comment:
Are you sure we should include these blue lamps for LRISr?

Reply:
I'm pretty sure they are all on the time. We need to get this information out of the headers eventually.